<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>newtask</class>
 <widget class="QWidget" name="newtask">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>615</width>
    <height>342</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>615</width>
    <height>342</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>615</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>266</x>
     <y>11</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>项目设置：</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>280</y>
     <width>441</width>
     <height>26</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_6">
    <item>
     <widget class="QPushButton" name="addButton">
      <property name="text">
       <string>新建</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="cancelbutton">
      <property name="text">
       <string>取消</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>260</width>
     <height>249</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>任务名称</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="tasknameEdit"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>任务描述：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="taskdetailEdit"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>112</y>
     <width>341</width>
     <height>26</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QCheckBox" name="weekdaytcheck">
      <property name="text">
       <string>每日提醒</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTimeEdit" name="remindtime"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>144</y>
     <width>341</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,2">
    <item>
     <widget class="QLabel" name="label_8">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>每周提醒时间</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="weekchose">
      <item>
       <widget class="QPushButton" name="w1">
        <property name="maximumSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton:checked{background-color:rgb(135,206,250); border-style: inset; }</string>
        </property>
        <property name="text">
         <string>一</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <property name="autoRepeat">
         <bool>false</bool>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="w2">
        <property name="maximumSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton:checked{background-color:rgb(135,206,250); border-style: inset; }</string>
        </property>
        <property name="text">
         <string>二</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <property name="autoRepeat">
         <bool>false</bool>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="w3">
        <property name="maximumSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton:checked{background-color:rgb(135,206,250); border-style: inset; }</string>
        </property>
        <property name="text">
         <string>三</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <property name="autoRepeat">
         <bool>false</bool>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="w4">
        <property name="maximumSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton:checked{background-color:rgb(135,206,250); border-style: inset; }</string>
        </property>
        <property name="text">
         <string>四</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <property name="autoRepeat">
         <bool>false</bool>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="w5">
        <property name="maximumSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton:checked{background-color:rgb(135,206,250); border-style: inset; }</string>
        </property>
        <property name="text">
         <string>五</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <property name="autoRepeat">
         <bool>false</bool>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="w6">
        <property name="maximumSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton:checked{background-color:rgb(135,206,250); border-style: inset; }</string>
        </property>
        <property name="text">
         <string>六</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <property name="autoRepeat">
         <bool>false</bool>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="w7">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="maximumSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton:checked{background-color:rgb(135,206,250); border-style: inset; }</string>
        </property>
        <property name="text">
         <string>日</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <property name="autoRepeat">
         <bool>false</bool>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>80</y>
     <width>341</width>
     <height>26</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>你打算每天做此任务多久</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="hourspinBox"/>
    </item>
    <item>
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>小时</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="minutespinBox"/>
    </item>
    <item>
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>分钟</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="datebox">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>23</y>
     <width>341</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="date">
    <item>
     <layout class="QHBoxLayout" name="start" stretch="1,2">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>开始日期</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateEdit" name="startdate">
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
        <property name="date">
         <date>
          <year>2023</year>
          <month>11</month>
          <day>1</day>
         </date>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="end" stretch="1,2">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>结束日期</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateEdit" name="enddate">
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
        <property name="date">
         <date>
          <year>2023</year>
          <month>12</month>
          <day>1</day>
         </date>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>180</y>
     <width>341</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_10">
      <property name="text">
       <string>完成状况</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="state0">
      <property name="text">
       <string>未开始</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
      <property name="autoExclusive">
       <bool>true</bool>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">statebuttonGroup</string>
      </attribute>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="state1">
      <property name="text">
       <string>进行中</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
      <property name="autoExclusive">
       <bool>true</bool>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">statebuttonGroup</string>
      </attribute>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="state2">
      <property name="text">
       <string>已完成</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
      <property name="autoExclusive">
       <bool>true</bool>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">statebuttonGroup</string>
      </attribute>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="remindlabel">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>260</y>
     <width>54</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: red;</string>
   </property>
   <property name="text">
    <string>remind</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="statebuttonGroup"/>
 </buttongroups>
</ui>
